### PR TITLE
Aqara Single Switch Module T1 (No Neutral)

### PR DESCRIPTION
**Pull request by Hongming6**

### CODE REVIEW

**Comment by Hongming6** [comment]
This PR corrects the English and Chinese names of some of the products that have been launched.

**Review by greens — COMMENTED** [review]
If you want to remove the `1` from the join names you will need to update the logic in this sub-driver: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/main/drivers/SmartThings/zigbee-switch/src/aqara/multi-switch/init.lua#L52

**Comment by Hongming6** [comment]
> If you want to remove the `1` from the join names you will need to update the logic in this sub-driver: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/main/drivers/SmartThings/zigbee-switch/src/aqara/multi-switch/init.lua#L52

Ok, I see the reason why the developer did that before, I restored such places and updated the labels with the switches inside the PR1290. please review it again

**Review by wkhenon — APPROVED** [review]

**Review by greens — APPROVED** [review]

File: tools/localizations/cn.csv [hunk 1]
```diff
@@ -58,6 +58,7 @@ Aqara Wireless Mini Switch T1,Aqara 无线开关 T1
 "Aqara Wireless Remote Switch E1 (Single Rocker)",Aqara 无线开关 E1 (贴墙式单键版)
 "Aqara LED Light Bulb T1 (Tunable White)",Aqara LED灯泡 T1 (可调色温)
 "Aqara Smart Wall Switch H1 EU (With Neutral, Single Rocker)",Aqara智能墙壁开关H1 EU (零火线单键版)
-"Aqara Smart Wall Switch H1 EU (With Neutral, Double Rocker)",Aqara智能墙壁开关H1 EU (零火线双键版)
+"Aqara Smart Wall Switch H1 EU (With Neutral, Double Rocker) 1",Aqara智能墙壁开关H1 EU (零火线双键版) 1
```
[the remark this answers is not in the document]
Comment (Hongming6) [reply tools/localizations/cn.csv]: okay，pls review again~

**Review by greens — APPROVED** [review]

**Pull request merged**